dev.docker-compose.yml

```diff
@@ -43,7 +43,7 @@ services:
         #     context: .
         #     dockerfile: Dockerfile.dev
         image: itechuw/openelis-global-2-dev:develop  
-        # platform: linux/amd64
+        platform: linux/amd64
```

Comment:
@reagan-meant you dont need to uncomment this. we publish mutiple platform image builds for the dev images.
just do a docker pull